kas.yml

```diff
@@ -31,6 +36,19 @@ repos:
     url: https://git.yoctoproject.org/git/meta-selinux
     refspec: 44d760413920ba440439b8bc7c2a71ca26cd7a2d
 
+  meta-secure-core:
+    url: https://github.com/jiazhang0/meta-secure-core
+    refspec: 628949305cdffff6798330d58e3d8c2bcf5c89f3
+    layers:
+      meta:
+      meta-tpm2:
+      meta-signing-key:
+      meta-efi-secure-boot:
+
+  meta-safeboot:
```

Comment:
I am also wondering if we want to have the safeboot by default in the each of the following images.

Reply:
The assumption was to provide the attestation for every TB image.

Reply:
Even for legacy? Doesn't it rely on UEFI Secure Boot?

Reply:
@macpijan Even for legacy. @miczyg1 Confirmed that.